Watch_List

```diff
@@ -0,0 +1,11 @@
+A card on the watchlist can be banned or unbanned and is there as a point of discussion 
```

Comment:
Would be nice if this was also Markdown format.

Reply:
Sure, can look at doing this conversion shortly